src/utils/cdk.ts

```diff
@@ -0,0 +1,266 @@
+import { resolve } from "pathe";
+import { Nitro } from "../types";
+import { writeFile } from "../utils";
+
+export async function generateCdkApp(nitro: Nitro) {
```

Comment:
This is way more verbose and error prone than SST, harder to tests and doesn't handle regular lambdas.